### PR TITLE
Adding a radius corner to card views

### PR DESCRIPTION
**Pull request by Guitouille**
SDK-389

### CODE REVIEW

File: navitiasdkux/src/main/res/drawable/background_rounded_corners.xml [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
-    <solid
-        android:color="@android:color/white" >
-    </solid>
-    <corners
-        android:radius="4dp">
-    </corners>
+    android:shape="rectangle">
+    <solid android:color="@android:color/white" />
+    <corners android:radius="@dimen/cards_corner_radius" />
 </shape>
```
Comment (EvenModiguy) [inline navitiasdkux/src/main/res/drawable/background_rounded_corners.xml]: Please add an empty line at the end of file.

File: navitiasdkux/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="cards_corner_radius">5dp</dimen>
+</resources>
```
Comment (EvenModiguy) [inline navitiasdkux/src/main/res/values/dimens.xml]: Please add an empty line at the end of file.

Comment (Guitouille) [reply navitiasdkux/src/main/res/values/dimens.xml]: Done

File: navitiasdkux/src/main/res/drawable/background_rounded_corners.xml (hunk 1 above)
Comment (Guitouille) [reply navitiasdkux/src/main/res/drawable/background_rounded_corners.xml]: Done

**Pull request merged**